spec/jobs/et_atos_export/claims_export_job_spec.rb

```diff
@@ -1,17 +1,18 @@
 require 'rails_helper'
```

Comment:
This is the spec for the job which moved from the application.  It also now injects the export service class instead of an instance of it - so it can be called multiple times with different export service configurations